Sources/TuistLoader/Loaders/ManifestLoader.swift

```diff
@@ -163,6 +163,10 @@ public class ManifestLoader: ManifestLoading {
             let manifestPath = path.appending(component: fileName)
             if !FileHandler.shared.exists(manifestPath) { continue }
             let data = try loadManifestData(at: manifestPath)
+            if Environment.shared.isVerbose {
+                let string = String(data: data, encoding: .utf8)
+                logger.debug("Could not load manifest represented by the following JSON representation:\n\(string)")
```

Comment:
@RomainBoulay I meant that the message shouldn't be "could not load", at this point we don't know if the JSON is valid or not. We could say:

```
Trying to load the manifest represented by the following JSON representation:\n\(string)
```

Reply:
Ha yeah, sorry I overlooked your previous comment 👍